quarkchain/cluster/root_state.py

```diff
@@ -665,4 +665,6 @@ def get_posw_info(
             block.header.hash_prev_block,
             self.db.get_root_block_header_by_hash,
         )
-        return get_posw_info(config, block.header, lambda: stakes, block_cnt, signer)
+        return get_posw_info(
+            config, block.header, lambda: stakes, block_cnt, signer=signer
```

Comment:
what is this change for?

Reply:
`get_posw_info` now has one more optional parameter, if `signer` is not specified, it would be regarded for the added `total_stakes_per_block` parameter

Reply:
Got it, thanks!